pyscriptjs/package.json

```diff
@@ -49,7 +49,7 @@
         "@codemirror/state": "6.1.2",
         "@codemirror/theme-one-dark": "6.1.0",
         "@codemirror/view": "6.3.0",
-        "codemirror": "6.0.1",
-        "toml-j0.4": "^1.1.1"
+        "@hoodmane/toml-j0.4": "^1.1.2",
```

Comment:
@vyings merged [my pull request](https://github.com/jakwings/toml-j0.4/pull/8) adding the type definitions to the toml-j0.4 distribution, and tagged a new version v1.1.2. However he says he has lost access to the npm registry account so I published v1.1.2 of tom-j0.4 here. This way we don't need to wget to patch up the types.